proto/forch_configuration.proto

```diff
@@ -165,6 +165,15 @@ message EventClientConfig {
  * Varz interface section in Forch configuration
  */
 message VarzInterfaceConfig {
-  // port number to curl for varz list
-  int32 varz_port = 1;
+  // port number to curl for forch varz list
+  int32 forch_varz_port = 1;
+
+  // port number to curl for faucet varz list
+  int32 faucet_varz_port = 2;
+
+  // port number to curl for gauge varz list
+  int32 gauge_varz_port = 3;
+
+  //port number for varz proxy
+  int32 proxy_varz_port = 4;
```

Comment:
nothing about this proxy is specific to varz, so it shouldn't be encoded in the names

Reply:
Addressed.